src/js/utils/dom.js

```diff
@@ -0,0 +1,7 @@
+export const $ = (selector, target = null) =>
+  target ? target.querySelector(selector) : document.querySelector(selector);
```

Comment:
```suggestion
export const $ = (selector, target = document) => target.querySelector(selector);
```
이렇게 간략하게 작성할 수 있을 것 같습니다 ㅎㅎ

Reply:
헉,,넘 깔끔해용... 😭